### PR TITLE
Add org.photoqt.PhotoQt

### PR DESCRIPTION
**Pull request by luspi**
PhotoQt is a different kind of image viewer. Website: https://photoqt.org

### CODE REVIEW

File: org.qt.photoqt.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+    "id": "org.qt.photoqt",
```
Comment (TingPing) [inline org.qt.photoqt.json]: `org.photoqt.PhotoQt`

File: org.qt.photoqt.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.10",
+    "sdk": "org.kde.Sdk",
+    "command": "photoqt",
+    "rename-desktop-file": "photoqt.desktop",
+    "rename-appdata-file": "photoqt.appdata.xml",
+    "rename-icon": "photoqt",
+    "finish-args": ["--share=ipc","--socket=x11","--socket=wayland","--filesystem=host","--device=dri","--socket=pulseaudio"],
```
Comment (TingPing) [inline org.qt.photoqt.json]: Please put these on their own line. Same applies for everything else and broken whitespace throughout.

File: org.qt.photoqt.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    "modules": [
+		{
+			"name": "exiv2",
+            "cmake": true,
```
Comment (TingPing) [inline org.qt.photoqt.json]: Use `"buildsystem": "cmake-ninja",`

File: org.qt.photoqt.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+            "builddir": true,
+			"sources": [ { "type": "archive", "url": "http://exiv2.org/exiv2-0.25.tar.gz", "sha256":"c80bfc778a15fdb06f71265db2c3d49d8493c382e516cb99b8c9f9cbde36efa4" } ]
+		},
+		{
+			"name": "libraw",
+            "cmake": false,
```
Comment (TingPing) [inline org.qt.photoqt.json]: That is the default, can be removed.

**Comment by TingPing** [comment]
bot, build org.photoqt.PhotoQt

File: org.qt.photoqt.json [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+    "sdk": "org.kde.Sdk",
+    "command": "photoqt",
+    "rename-desktop-file": "photoqt.desktop",
+    "rename-appdata-file": "photoqt.appdata.xml",
+    "rename-icon": "photoqt",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=host",
```
Comment (nedrichards) [inline org.qt.photoqt.json]: this seems like too broad a privilege request

**Comment by nedrichards** [comment]
The JSON file needs to be the same as the app-id and org.qt.photoqt.json isn't org.photoqt.PhotoQt so the build fails https://flathub.org/builds/#/builders/76/builds/21

File: org.qt.photoqt.json (hunk 5 above)
Comment (Sesivany) [reply org.qt.photoqt.json]: Hi, I'm the one who updated manifest to be Flathub ready. I'm not sure PhotoQt can work in full sandbox via portal, so there needs to be direct access to the filesystem. Access only to home would be fine? The original manifest had "host" and I thought the app author wanted it to browse outside home as well, so I kept it there.

Comment (nedrichards) [reply org.qt.photoqt.json]: @luspi what's the expected use case here? xdg-pictures and videos? home? viewing random assets from the rest of the system?

Comment (luspi) [reply org.qt.photoqt.json]: I think "viewing random assets" is a pretty good description of its use has. It includes of course folders like xdg-pictures and home, but also goes beyond that to anywhere a use might encounter an image of some type...

**Comment by nedrichards** [comment]
bot, build org.photoqt.PhotoQt

**Comment by nedrichards** [comment]
Looks like there's something wrong in the <release> section:

> Error loading AppData file: failed to parse /app/share/appdata/org.photoqt.PhotoQt.appdata.xml: Error on line 81 char 6: <releases> already set '​' and tried to replace with '  
​ '
https://flathub.org/builds/#/builders/4/builds/2195 e.g.

**Comment by Sesivany** [comment]
I copied that section from a different file and it has probably introduced some non-visible characters that shouldn't be there. When I removed all the spaces, the file gets validated. I also noted that if <release> doesn't have any nested elements, which it doesn't in this case because I didn't include download/installed sizes, it can self-close. So that part can be simplified into this:
<releases>
<release version="1.6" date="2018-02-20" />
</releases>

**Comment by luspi** [comment]
Simplified release section, and found the "faulty" spaces in the appdata file. It should (hopefully) work now!

**Comment by nedrichards** [comment]
bot, build org.photoqt.PhotoQt

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/org.photoqt.PhotoQt

**Pull request closed without merging**